lab-bundle-spec.md

```diff
@@ -264,6 +277,34 @@ No additional attributes
 > **NOTE:** A draft of the `aws-account` resource type was previously specified
 > in this document. See [previous version](https://github.com/CloudVLab/qwiklabs-content-bundle-spec/blob/93896ced4ae5b543132d7a10d838ac17bd5ae3e1/lab-bundle-spec.md) for details.
 
+#### Display Outputs
```

Comment:
@reviewers  - I'm not a huge fan of the name "Display Outputs". I'm trying to evoke something closer to the idea of "Givens" in a problem set.

Any suggestions would be appreciated.

Reply:
Some ideas:
- student_facing
- provided
- surfaced
- public

Reply:
I really like 'student_facing', because it's clear about who's perspective it's taking.

But I worried about 'facing' being to idiomatic, so I went with 'student_visible'. If anyone thinks I'm being too sensitive, let me know. At a gut level I definitely prefer 'student_facing'.